eng/targets/Wix.Common.props

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
     <ProductVersion>3.14</ProductVersion>
-    <WixVersion>3.14.1.8722</WixVersion>
```

Comment:
FYI the 3.14.1.8722 is the version used for the zip, the full version you have in this change is for the nupkg.  If you want the nupkg, then you can use the Arcade property `MicrosoftSignedWixVersion` for that

Reply:
Thanks, updated